content/en/docs/Configuration/kialis.kiali.io.md

```diff
@@ -326,16 +326,19 @@ spec:
       dashboards:
       - name: "Istio Service Dashboard"
         variables:
+          datasource: "var-datasource"
```

Comment:
You do not edit this file - its automatically generated from the cr example in the operator repo when we release. Revert all changes to this file.